package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "./node_modules/.bin/eslint ./*.js app config lib server tests",
     "lint:hbs": "ember-template-lint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "ember test --silent --reporter dot"
```

Comment:
did this work? lemme know, great if it did! what does each statement do?

Reply:
`--silent` supposedly supresses top-level warnings and `to silence unwanted output`, but it doesn't necessarily do so for warnings stemming from addons and plugins? `--reporter dot` represents tests passing/failing using this dot notation:

<img width="562" alt="Screen Shot 2019-10-21 at 5 54 50 PM" src="https://user-images.githubusercontent.com/3311663/67246390-5434de00-f42c-11e9-9929-67069c35accf.png">

I'm thinking I should actually change it back for now, since it's useful to have it be a little verbose.